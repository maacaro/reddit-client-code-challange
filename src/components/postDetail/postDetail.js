import React from "react";

export const PostDetail = ({ title, imageURL, author }) => {
  return (
    <div>
      <h3>{author}</h3>
      <img src={imageURL} alt="" />
      <p>{title}</p>
    </div>
  );
};
