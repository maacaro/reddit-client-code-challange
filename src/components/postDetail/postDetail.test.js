import React from "react";
import { PostDetail } from "./postDetail";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <PostDetail title="Post title" imageURL="url" author="Manuel Castro" />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
