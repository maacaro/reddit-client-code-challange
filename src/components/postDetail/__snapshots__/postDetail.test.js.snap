// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h3>
    Manuel Castro
  </h3>
  <img
    alt=""
    src="url"
  />
  <p>
    Post title
  </p>
</div>
`;
